training/provision/playbook.yml

```diff
@@ -15,16 +15,51 @@
   - name: Gather facts for first time
     ansible.builtin.setup:
 
-  - name: Check Podman Present
-    ansible.builtin.package:
-      name: podman
+  - name: Required packages
+    ansible.builtin.dnf:
+      name:
+      - https://s3.eu-west-2.amazonaws.com/amazon-ssm-eu-west-2/latest/linux_amd64/amazon-ssm-agent.rpm
```

Comment:
I'm curious why...this beast has deep hooks into everything, it's Amazon wanting to own systems management.

Reply:
I use it for connecting to the instance for debugging purposes. It will be removed as soon as the workflow starts working